src/Test/Utilities/Portable/LazyToString.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Roslyn.Test.Utilities
+{
+    internal sealed class LazyToString
```

Comment:
This is far too much magic to not get a comment.

Reply:
Removed this type